server_spidercast/src/MCC_RUM/src/rumFireOutThread.c

```diff
@@ -1603,7 +1603,7 @@ THREAD_RETURN_TYPE FireOutThread(void *param)
         if ( rmmTRec[inst]->FO_Thrds[i] == THREAD_RUNNING ||
           rmmTRec[inst]->FO_Thrds[i] == THREAD_KILL )
           done = 0;
-        
+          
```

Comment:
This doesn't look like you've fixed this section. I don't think the llmSimpleTraceInvoke should be in the if (compare with the similar section a few lines up)

Reply:
Oh, the indentation fix didn't reflect. Will fix this right away.

Reply:
Strange addition of whitespace on line 1606 - but that wouldn't block the merge

Reply:
Removed the whitespace.